backend/server/api/services/todos.service.js

```diff
@@ -18,6 +18,17 @@ export class ToDoService {
     const newTodo = allTodos.child(newTodoRef.key).toJSON();
     return { ...newTodo, key: newTodoRef.key };
   }
+
+  async all() {
+    const allTodos = await this.db.ref("todo").once("value");
+    return allTodos;
+  }
+
+  async byId(id) {
+    const todo = await this.db.ref("todo/" + id).once("value");
```

Comment:
You have to convert this firebase data to a Javascript object. See the other methods.

Reply:
Done